core/src/main/java/nl/nn/adapterframework/jdbc/migration/DatabaseMigratorBase.java

```diff
@@ -135,4 +135,18 @@ public final ApplicationContext getApplicationContext() {
 	public boolean isEnabled() {
 		return AppConstants.getInstance(configuration.getClassLoader()).getBoolean("jdbc.migrator.active", false);
 	}
+
+	/**
+	 * Checks whether the configuration contains liquibase script that can be translated into sql statements in the classpath
+	 */
+	public boolean hasLiquibaseScript(Configuration config) {
+		AppConstants appConstants = AppConstants.getInstance(config.getClassLoader());
+		String changeLogFile = appConstants.getString("liquibase.changeLogFile", "DatabaseChangelog.xml");
+		LiquibaseResourceAccessor resourceAccessor = new LiquibaseResourceAccessor(config.getClassLoader());
+		if(resourceAccessor.getResource(changeLogFile) == null) {
+			log.debug("No liquibase script ["+changeLogFile+"] found in the classpath of configuration ["+config.getName()+"]");
+			return false;
+		}
+		return true;
+	}
```

Comment:
Can you then please override `isEnabled` in LiquibaseMigrator and call the method `getChangeLogFile`? The method returns a FNFE, I think the easiest is to catch the exception and return false. That way the isEnabled works uniformly, and there is no duplicate code.